stdlib/stdlib.mli

```diff
@@ -1223,6 +1264,104 @@ val at_exit : (unit -> unit) -> unit
    The functions are called in 'last in, first out' order: the
    function most recently added with [at_exit] is called first. *)
 
+(** {1:precedence Appendix: precedence level and associativity of operators }
```

Comment:
Maybe have an implementation comment here indicating to remember to update the other table (in the manual), and also the other way around?

Reply:
I added the two synchronization comments.